src/sonic-config-engine/sonic-cfggen

```diff
@@ -317,9 +317,6 @@ def main():
         if args.port_config is None:
             args.port_config = device_info.get_path_to_port_config_file(hwsku)
         (ports, _, _) = get_port_config(hwsku, platform, args.port_config, asic_id)
-        if not ports:
-            print('Failed to get port config', file=sys.stderr)
-            sys.exit(1)
         deep_update(data, {'PORT': ports})
```

Comment:
>ports [](http://example.com/codeflow?start=35&length=5)

Does it work with:
1. ports == None
2. Or, ports == {}

Reply:
@qiluo-msft 
Thanks, I changed the code according to your comment, I will explain.

The code I deleted verified both conditions:
1. ports is not None.
2. ports is not empty

Since now (in modular chassis without linecards), ports dictionary can be empty, we just need to verify it is not None.